sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1121,6 +1124,68 @@ public void StartRecognizeBusinessCardsFromUriThrowsForNonExistingContent()
             Assert.AreEqual("FailedToDownloadImage", ex.ErrorCode);
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false, Ignore ="File not on Github")]
+        public async Task StartRecognizeBusinessCardsCanParseMultipageForm(bool useStream)
+        {
+            var client = CreateFormRecognizerClient();
+            var options = new RecognizeBusinessCardsOptions() { IncludeFieldElements = true };
+            RecognizeBusinessCardsOperation operation;
+
+            if (useStream)
+            {
+                using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.BusinessMultipage);
+                using (Recording.DisableRequestBodyRecording())
+                {
+                    operation = await client.StartRecognizeBusinessCardsAsync(stream, options);
+                }
+            }
+            else
+            {
+                var uri = FormRecognizerTestEnvironment.CreateUri(TestFile.BusinessMultipage);
+                operation = await client.StartRecognizeBusinessCardsFromUriAsync(uri, options);
+            }
+
+            RecognizedFormCollection recognizedForms = await operation.WaitForCompletionAsync(PollingInterval);
```

Comment:
out of curiosity, since I think this code works: where is the `PollingInterval` variable defined? Is this a .net thing I'm not familiar with, I'm not seeing it defined anywhere in the test file, but a bunch of tests are using it with no problem

Reply:
Yes. We have it in the `FormRecognizerLiveTestBase` which this class inherits from. 
The [PollingInterval](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerLiveTestBase.cs#L14) is our way to tell the tests that if we are in `Playback` mode don't wait.